src/components/sections/user-sync-test/run-test.js

```diff
@@ -44,7 +45,7 @@ const RunTest = ({ states }) => {
                                         {test.title}
                                     </p>
                                     <p className={itemStyles.subItemItem}>
-                                        {i18n.t('Recommended maximum:')}{' '}
+                                        {i18n.t('Recommended maximum:')}
```

Comment:
This should use an i18next interpolation variable (`i18n.t('Recommended maximum: {{maxValue}}', { maxValue: test.maxValue })`)

Reply:
Hi Austin! When I use interpolation I get this:

![image](https://user-images.githubusercontent.com/29384664/94493823-45fe4e80-01b3-11eb-8084-66cc859ace11.png)

Reply:
Sorry @Sharmyn28, my fault!  This is due to the `:` character being used as the namespace separator.  You can avoid this with (I believe):

```js
i18n.t('Recommended maximum: {{maxValue}}', { nsSeparator: '---', maxValue: maxValue })
```

You can confirm it works by making sure `Recommended maximum: {{maxValue}}` shows up in the `en.pot` file

Reply:
Thanks, this did the trick, I checked the `en.pot `